internal/pkg/cli/app_init.go

```diff
@@ -53,11 +53,21 @@ type initAppOpts struct {
 	prompt               prompter
 	prog                 progress
 	iam                  policyLister
+	iamRoleManager       roleManager
 	isSessionFromEnvVars func() (bool, error)
 
 	cachedHostedZoneID string
 }
 
+type errAppAlreadyExistsInAccount struct {
```

Comment:
Can we move the definition of the error struct as well? To just above their method definition (`Error` and `RecommendedActions`)